<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerMainWindow</class>
 <widget class="QMainWindow" name="PlayerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KylinPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="singerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>歌手:</string>
    </property>
   </widget>
   <widget class="QLabel" name="albumLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑:</string>
    </property>
   </widget>
   <widget class="QLabel" name="songLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>歌名:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="channelComboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>314</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>本地歌曲</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>网络歌曲</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="playTimeSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
selection-background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/prev.png</normaloff>:/images/prev.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
selection-background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
selection-background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="volumeDial">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>40</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>3.700000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="artWorkLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>240</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>封面图</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="mediaMenu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="localAction"/>
    <addaction name="netAction"/>
   </widget>
   <widget class="QMenu" name="aboutMenu">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>收藏</string>
    </property>
    <addaction name="addFavAction"/>
    <addaction name="playFavAction"/>
   </widget>
   <addaction name="mediaMenu"/>
   <addaction name="menu"/>
   <addaction name="aboutMenu"/>
  </widget>
  <action name="localAction">
   <property name="text">
    <string>本地歌曲</string>
   </property>
  </action>
  <action name="netAction">
   <property name="text">
    <string>网络歌曲</string>
   </property>
  </action>
  <action name="addFavAction">
   <property name="text">
    <string>加入收藏</string>
   </property>
  </action>
  <action name="playFavAction">
   <property name="text">
    <string>播放收藏</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于KylinPlayer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
